src/auto/user_content_filter.rs

```diff
@@ -1,31 +0,0 @@
-// This file was generated by gir (https://github.com/gtk-rs/gir)
```

Comment:
Any idea what caused this file to be removed?

Reply:
That file was already disabled in the gir file, I just forgot to remove it :) It could be re-enabled with some manual work

Reply:
Its mostly `make regen_check` test that was added to CI that made me notice it :)